packages/core/src/components/ozon-content/ozon-content.scss

```diff
@@ -115,6 +115,42 @@ span[role="paragraph"] {
   }
 }
 
+.dso-ozon-lijst {
+  ul {
+    &.expliciet {
+      list-style: none;
+      padding-inline-start: 0;
```

Comment:
Ik zou er wel voor opteren om padding-inline-start te handhaven. En dan met `position: absolute;` het lijstnummer in die padding te plaatsen. Dan hoef je niet meer aan de andere content van een `<Li>` te zitten kwa positionering. En gaan geneste lijsten altijd goed.